apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,105 @@
+/* global PlayerView */
+
+'use strict';
+
+require('/js/ui/views/player_view.js');
+require('/shared/js/l10n.js');
+
+suite('Star Rating Bar Accessibility Test', function () {
+
+    var pv,
+    ratings;
+
+    suiteSetup(function () {
+       
```

Comment:
Just for clarity, you're talking about the empty lines at various places throughout the file, right?  If so, do you have a site I could reference for proper (blank) line spacing ?  I've seen a variety of styles throughout the files and I'm a little unsure which one to go with.